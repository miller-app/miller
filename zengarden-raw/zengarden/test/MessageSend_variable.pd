#N canvas 333 115 441 428 10;
#X obj 95 63 loadbang;
#X obj 95 93 metro 1000;
#X obj 176 316 r send-test;
#X obj 95 203 s send-test;
#X obj 176 339 print received;
#X msg 95 173 1;
#X msg 171 204 \; send-test \$1 \;;
#X msg 171 173 2;
#X obj 274 252 s;
#X obj 301 227 symbol;
#X obj 274 173 t b f;
#X msg 301 199 send-test;
#X msg 274 215 3;
#X obj 95 123 t b b b;
#X connect 0 0 1 0;
#X connect 1 0 13 0;
#X connect 2 0 4 0;
#X connect 5 0 3 0;
#X connect 7 0 6 0;
#X connect 9 0 8 1;
#X connect 10 0 12 0;
#X connect 10 1 11 0;
#X connect 11 0 9 0;
#X connect 12 0 8 0;
#X connect 13 0 5 0;
#X connect 13 1 7 0;
#X connect 13 2 10 0;
