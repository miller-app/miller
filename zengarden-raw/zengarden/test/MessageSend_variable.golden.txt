[@ 0.000ms] received: 3
[@ 0.000ms] received: 2
[@ 0.000ms] received: 1